salt/modules/win_service.py

```diff
@@ -817,6 +817,7 @@ def enable(name, **kwargs):
 
     Args:
         name (str): The name of the service to enable.
+        **kwargs supports StartType
```

Comment:
This should probably add two named arguments here. `start_type` with a default of `auto` and `start_delayed` with a default of `False`. To conform with other parameter names in salt please avoid CamelCase.

There needs to be additional documentation here as well for both parameters... similar to that for `modify`.